pkg/controller/controller.go

```diff
@@ -558,6 +558,14 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 	return pv, nil
 }
 
+func getReclaimPolicy(sc *storagev1.StorageClass) v1.PersistentVolumeReclaimPolicy {
+	reclaim := v1.PersistentVolumeReclaimDelete
```

Comment:
I don't think we should default reclaim policy if nothing is set in the StorageClass.  If it's not set, then we should not fill it in, and let api server default it.

Reply:
done